source/html/renderer/src/card-elements.ts

```diff
@@ -839,6 +839,7 @@ export class ToggleInput extends Input {
 export class Choice {
     title: string;
     value: string;
+    isSelected: boolean;
```

Comment:
I found the isSelected property in the Docs: http://adaptivecards.io/explorer/#InputChoiceSet
The docs do not mention ChoiceSet as having a value.

Do the docs need to be updated?